File: client/iih_stub/iih.ci_stub/iih.ci.ord/cfg/dto/msg/d/MsgType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.cfg.dto.msg.d
{
   public enum MsgType {
	/// <summary>
	/// 消息
	/// </summary>
    [Description("消息")]
    INFO=0,
	/// <summary>
	/// 错误
	/// </summary>
    [Description("错误")]
    ERROR=1,
	/// <summary>
	/// 询问确认
	/// </summary>
    [Description("询问确认")]
    CONFIRM=2,
   }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/cfg/dto/msg/d/MsgType.java
package iih.ci.ord.cfg.dto.msg.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface MsgType {

    @DmEnumDesc(name="消息",description="消息")
	public static final Integer INFO=0; //消息
    @DmEnumDesc(name="错误",description="错误")
	public static final Integer ERROR=1; //错误
    @DmEnumDesc(name="询问确认",description="询问确认")
	public static final Integer CONFIRM=2; //询问确认
}	
